### DISCUSSION TITLE
Share details on figure in readme

### DISCUSSION THREAD

**Discussion by julienguegan** [peterdsharpe/AeroSandbox#65] [Q&A] [answered]
Hello @peterdsharpe  ! 

I would like to reproduce the figures in the readme (anel1_naca4412.png), do you think you could share the details on how to generate it ?

Thank you a lot for your amazing work !

**Comment by peterdsharpe** [comment, marked as answer]
Hi there - apologies for the delay, but please see [this tutorial](https://github.com/peterdsharpe/AeroSandbox/blob/master/tutorial/06%20-%20Aerodynamics/01%20-%20AeroSandbox%203D%20Aerodynamics%20Tools/01%20-%20Vortex%20Lattice%20Method/01%20-%20Vortex%20Lattice%20Method.ipynb) for details on how to interface with the ASB aerodynamics methods.
